tools/run-tests-within-container

```diff
@@ -8,7 +8,7 @@ TESTS_FAILED_FLAG=0
 run_test() {
     # Allow command line options without quotes
     # shellcheck disable=SC2086
-    prove -l ${PROVE_ARGS} "$@" || TESTS_FAILED_FLAG=1
+    prove -l ${PROVE_ARGS:+} "$@" || TESTS_FAILED_FLAG=1
```

Comment:
What about the line above now?

Reply:
Here I asked you about the line above, the `# shellcheck disable=SC2086`.

Reply:
as line above I thought you meant the other `prove` command. checking again